images/flexvolume/rookflexvolume.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
I think we should replace this with a new "proxy" daemon verb in cmd/rook that install the flex volume on startup then becomes our new proxy server listening on an api endpoint. let's discuss the design.